1. Basics/task_6/hw6_counter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * File name, which contains the counter
+ */
+define("FILE_NAME", "counter.txt");
+
+/**
+ * Counter start value
+ */
+define("COUNTER_START_VALUE", 0);
+
+if(!file_exists(FILE_NAME)) {
+    $file = fopen(FILE_NAME, "w") or die("I can't create a file");
```

Comment:
better not to use `or`, such behavior can confuse other developers

Reply:
you can use just `file_put_contents`

Reply:
👌 Done.